tests/runtime/call

```diff
@@ -488,6 +488,13 @@ REQUIRES_FEATURE dpath kfunc
 TIMEOUT 5
 AFTER ./testprogs/syscall read
 
+NAME path_with_optional_size
+RUN {{BPFTRACE}} -ve 'kfunc:filp_close { if (!strncmp(path(args.filp->f_path, 48), "tmp/bpftrace_runtime_test_syscall_gen_read_temp", 48)) { printf("OK\n"); exit(); } }'
```

Comment:
Sorry this one is my fault. I didn't look at this test close enough. I think we need to add an additional size check on the string returned by path because right now we're not testing if the shorter size is actually used as `strncmp` is also clamping e.g.

```suggestion
RUN {{BPFTRACE}} -ve 'kfunc:filp_close { $p = path(args.filp->f_path, 48);  if ( sizeof($p) == 48 && !strncmp($p, "tmp/bpftrace_runtime_test_syscall_gen_read_temp", 48)) { printf("OK\n"); exit(); } }'
```